src/server.ts

```diff
@@ -398,6 +403,11 @@ export function makeServer<E = unknown>(options: ServerOptions<E>): Server<E> {
     opened(socket, extra) {
       if (socket.protocol !== GRAPHQL_TRANSPORT_WS_PROTOCOL) {
         socket.close(1002, 'Protocol Error');
+        logDeveloperWarning(
+          `Invalid protocol: '${socket.protocol}'. 
+          Only ${GRAPHQL_TRANSPORT_WS_PROTOCOL} is supported.
+          For more details see https://github.com/enisdenjo/graphql-ws/issues/83`,
+        );
```

Comment:
No need to have this `logDeveloperWarning`. I try avoiding any logs and rely on exposing as much as possible for the users. Simply put the `if != PROD console.warn` right here.

Also, the indentation in backticks will be preserved. Consider inlining the message.